server.js

```diff
@@ -64,7 +64,8 @@ server.use((req, res, next) => {
           console.log(
             `Error: ${err},
             Request: ${req.url},
-            Host: ${req.get('host')},
+            IP: ${req.ip},
+            Host: ${req.hostname},
```

Comment:
Label 'Host' on $ statement.<br>Expected an assignment or function call and instead saw an expression.<br>Too many errors. (59% scanned).